straxen/get_corrections.py

```diff
@@ -37,72 +38,50 @@ def correction_options_wrapped(run_id, conf, *arg):
-        if not isinstance(model_conf, tuple) or len(model_conf) != 2:
-            # Raise a value error if the condition is not met. We should have:
-            # ("CMT_model", -> To specify that we want to use the online
-            #                  corrections management tool
-            #   (
-            #   "to_pe_model", -> This is to specify that we want  the gains
-            #   "v1", -> The version of the correction 'v1' is the online version
-            #   )
-            raise ValueError('CMT gain model should be similar to:'
-                             '("CMT_model", ("to_pe_model", "v1"). Instead got:'
-                             f'{model_conf}')
-        # is this the best way to do this?
-        is_nt = n_pmts == straxen.n_tpc_pmts or n_pmts == straxen.n_nveto_pmts or n_pmts == straxen.n_mveto_pmts
-
-        corrections = straxen.CorrectionsManagementServices(is_nt=is_nt)
-        to_pe = corrections.get_corrections_config(run_id, model_conf)
-
-        return to_pe
-
-    elif model_type == 'to_pe_per_run':
-        warn("to_pe_per_run will be replaced by CorrectionsManagementSevices",
-             DeprecationWarning, 2)
-        # Load a npy file specifying a run_id -> to_pe array
-        to_pe_file = model_conf
-        x = straxen.get_resource(to_pe_file, fmt='npy')
-        run_index = np.where(x['run_id'] == int(run_id))[0]
-        if not len(run_index):
-            # Gains not known: using placeholders
-            run_index = [-1]
-        to_pe = x[run_index[0]]['to_pe']
-
-    elif model_type == 'to_pe_constant':
-        if model_conf in FIXED_TO_PE:
-            return FIXED_TO_PE[model_conf]
-
-        try:
-            # Uniform gain, specified as a to_pe factor
-            to_pe = np.ones(n_pmts, dtype=np.float32) * model_conf
-        except np.core._exceptions.UFuncTypeError as e:
-            raise (str(e) +
-                   f"\nTried multiplying by {model_conf}. Insert a number instead.")
-    else:
-        raise NotImplementedError(f"Gain model type {model_type} not implemented")
+def get_to_pe(run_id, conf, n_pmts):
+    """
+    To pe values for TPC, n-veto mu-veto PMTs, this is also now as gains
+    :param run_id: run id from runDB
+    :param conf: configuration 
+    :n_pmts: number of PMTs
+    :return: array of to_pe values
+    """
+    if isinstance(conf, tuple) and len(conf) == 3:
+        is_nt = conf[-1]
+        model_conf, global_version = conf[:2]
+        if 'to_pe_model' in model_conf:
+            corrections = straxen.CorrectionsManagementServices(is_nt=is_nt)
+            to_pe = corrections.get_corrections_config(run_id, conf[:2])
+
+            return to_pe
+        else:
+            raise ValueError("Wrong configuration. "
+                         "Please use the following format: "
+                         "(config->str, model_config->str or number, is_nT->bool) "
+                         f"User specify {conf} please modify")
```

Comment:
This is more of less identical with first part of `get_correction_from_cmt`

Reply:
you are right, I remove `get_to_pe()`